src/twisted/mail/smtp.py

```diff
@@ -919,6 +931,8 @@ class SMTPClient(basic.LineReceiver, policies.TimeoutMixin):
     # None, perform no timeout checking.
     timeout = None
 
+    _encoding = "ascii"
+
     def __init__(self, identity, logsize=10):
         if isinstance(identity, unicode):
             identity = identity.encode('ascii')
```

Comment:
Should be `self._encoding`, not `"ascii"`.

Reply:
ok